metadata.rb

```diff
@@ -1,14 +1,25 @@
 name              'resolver'
-maintainer        'Chef Software, Inc.'
-maintainer_email  'cookbooks@chef.io'
+maintainer        'Sous Chefs'
+maintainer_email  'help@sous-chefs.org'
 license           'Apache-2.0'
 description       'Configures /etc/resolv.conf via attributes'
-version '2.1.0'
+version           '2.1.0'
+source_url        'https://github.com/sous-chefs/resolver'
+issues_url        'https://github.com/sous-chefs/resolver/issues'
+chef_version      '>= 12.15'
 
-%w(aix ubuntu debian fedora centos redhat oracle scientific amazon freebsd openbsd mac_os_x solaris2 opensuse opensuseleap suse).each do |os|
-  supports os
-end
-
-source_url 'https://github.com/chef-cookbooks/resolver'
-issues_url 'https://github.com/chef-cookbooks/resolver/issues'
-chef_version '>= 12.15'
+supports          'ubuntu'
+supports          'debian'
+supports          'fedora'
+supports          'centos'
+supports          'redhat'
+supports          'oracle'
+supports          'scientific'
+supports          'amazon'
+supports          'freebsd'
+supports          'openbsd'
+supports          'mac_os_x'
+supports          'solaris2'
+supports          'opensuse'
+supports          'opensuseleap'
+supports          'suse'
```

Comment:
Please order these supports alphabetically and also change the indentation to the following for them:
```suggestion
supports 'suse'
```